### PR TITLE
Update to newer images of components used by BKPR.

### PR DESCRIPTION
**Pull request by falfaro**

### CODE REVIEW

File: manifests/components/versions.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    "kibana": "bitnami/kibana:5.6.12-r15",
-    "nginx-ingress-controller": "bitnami/nginx-ingress-controller:0.21.0-r0",
+    "cert-manager": "bitnami/cert-manager:0.5.2-r37",
+    "external-dns": "bitnami/external-dns:0.5.9-r19",
+    "elasticsearch": "bitnami/elasticsearch:5.6.14-r2",
+    "fluentd": "bitnami/fluentd:1.3.2-r13",
+    "grafana": "bitnami/grafana:5.4.2-r12",
+    "kibana": "bitnami/kibana:5.6.14-r11",
+    "nginx-ingress-controller": "bitnami/nginx-ingress-controller:0.21.0-r12",
     "oauth2_proxy": "bitnami/oauth2-proxy:0.20180625.74543-debian-9-r6",
```
Comment (sameersbn) [inline manifests/components/versions.json]: The latest available version for oauth2-proxy image is `bitnami/oauth2-proxy:0.20180625.74543-r107`.

File: manifests/components/versions.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-    "prometheus": "bitnami/prometheus:2.4.3-r31"
+    "prometheus": "bitnami/prometheus:2.5.0-r41"
```
Comment (sameersbn) [inline manifests/components/versions.json]: Why not upgrade to `2.6.0` series?
Also, we're missing the listing for `alertmanager` and `configmap-reload`.

File: manifests/components/versions.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
```
Comment (sameersbn) [inline manifests/components/versions.json]: I suggest renaming this file from `versions.json` to `images.json`

Comment (sameersbn) [reply manifests/components/versions.json]: Also sorting the images by name would help manual lookup, wdyt?

File: manifests/components/versions.json (hunk 1 above)
Comment (falfaro) [reply manifests/components/versions.json]: Right. Done.

File: manifests/components/versions.json (hunk 2 above)
Comment (falfaro) [reply manifests/components/versions.json]: Done.

**Pull request merged**